packages/inter-protocol/src/proposals/deploy-price-feeds.js

```diff
@@ -221,6 +239,14 @@ export const deployPriceFeeds = async (powers, config) => {
     priceAggregatorRef.bundleID,
   );
 
+  // if retiredContractInstances doesn't exist, create it.
+  const { retiredContractInstances: retiredInstanceWriter } = powers.produce;
+  const contractInstanceMap = makeScalarBigMapStore(
+    'retiredContractInstances',
+    { durable: true },
+  );
+  retiredInstanceWriter.resolve(contractInstanceMap);
```

Comment:
most evals say bind `produce.someName` as `produceSomeName`. please be consistent with that. and no need to bind the map locally.
```suggestion
  const produceRetiredContractInstances = powers.produce.retiredContractInstances;
  // first to produce wins so this can be attempted concurrently with other producers
  produceRetiredContractInstances.resolve(makeScalarBigMapStore(
      'retiredContractInstances',
      { durable: true },
    );
  );
```
```

Reply:
done